features/gba-dev/ouders.feature

```diff
@@ -0,0 +1,234 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Ouders van een persoon raadplegen
+  Van een persoon kunnen ouders worden geleverd.
+  Dit bevat gegevens over de familierechtelijke betrekking plus enkele identificerende eigenschappen van de ouder.
+
+@gba
+Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie 3 heeft ouderAanduiding "2"
+
+  Scenario: Persoon heeft twee ouders
+    Gegeven de persoon met burgerservicenummer '000000152' heeft een ouder '1' met de volgende gegevens
+    | voornamen (02.10)  |
+    | Lisette            |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+    | voornamen (02.10)  |
+    | Guîllaumé          |
+    Als personen wordt gezocht met de volgende parameters
```

Comment:
Hier zou moeten staan 'Als gba personen wordt gezocht met de volgende parameters'

Reply:
Is aangepast.